src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -141,13 +144,22 @@
           :sub-label nil
           :chevron?  false}))
 
-(defn notifications-entry []
-  (entry {:icon      :i/notifications
-          :label     (i18n/label :t/notifications)
+(defn edit-name-image-entry []
+  (entry {:icon      :i/edit
+          :label     (i18n/label :t/edit-name-and-image)
           :on-press  #(js/alert "TODO: to be implemented, requires design input")
```

Comment:
Could you add your username to these TODOs? Adding your username can help other people to know who's the person with the original context.